### PR TITLE
Custom DNA - Batch 1

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Added **31** custom DNA entries for the enclosed characters:

<details><summary>Click to expand</summary>

**Orcs**
* Gul'dan _(base by @Garthor19, adjusted by @ercarp)_
* Orgrim _(by @Garthor19)_
* Garona _(by @ercarp)_
* Blackhand _(base by @Garthor19, adjusted by @ercarp)_
* Durotan _(by @ercarp)_
* Go'el/Thrall _(base by @Garthor19, adjusted by @ercarp)_
* Kilrogg _(by @Garthor19)_

**Dwarves**
* Magni Bronzebeard _(by @ercarp)_

**Elves**
* Anasterian Sunstrider _(by @ercarp)_
* Kael'thas Sunstrider _(by @ercarp)_
* Lor'themar Theron _(by @ercarp and @TheDrPibb1999)_
* Sylvanas Windrunner _(by @ercarp)_
  * **Note**: Sylvanas's undead form uses custom make-up in addition to a scripted undead appearance. Her appearance is also currently affected by the **_Beautiful_** trait, which is not intended.

**Trolls**
* Vol'jin _(by @ercarp)_
  * **Note**: Vol'jin's portrait uses custom warpaint when he is aged 18 or above.

**Humans**
* Tirion Fordring _(by @Garthor19 and @ercarp)_
* Aiden Perenolde _(by @ercarp)_
* Medivh _(by @ercarp)_
* Llane Wrynn _(by @ercarp)_
* Varian Wrynn _(by @ercarp)_
* Anduin Wrynn _(by @ercarp)_
* Thoras Trollbane _(by @ercarp)_
* Daelin Proudmoore _(by @ercarp)_
* Katherine Proudmoore _(by @ercarp)_
* Genn Greymane _(by @ercarp)_
* Terenas II Menethil _(by @ercarp)_
* Arthas Menethil _(by @Garthor19 and @ercarp)_
  * **Note**: There are two versions of Arthas's DNA, one made by me and another made by Garthor19. Vote on which one you prefer:
<details><summary>Click to expand</summary>

Артха

</details>

* Antonidas _(by @ercarp)_
  * **Note**: Antonidas's beard turns white when aged 50 or above.
* Uther the Lightbringer _(by @ercarp)_
  * **Note**: Uther's portrait uses a custom hair and beard when he is aged 40 or above.
* Turalyon _(by @ercarp)_
* Anduin Lothar _(by @ercarp)_
  * **Note**: Lothar has a portrait modifier that changes his beard and hairstyle when he is aged 50 or above.
* Khadgar _(by @ercarp)_
* Tirion Fordring _(by @Garthor19)_
* Nathanos Marris _(by @ercarp)_
  * **Note**: Nathanos uses a scripted undead appearance based on his post-Dark Mirror look.
* Othmar Garithos _(by @ercarp)_
  * **Note**: Garithos turns bald when aged 30 or above.

</details>

This batch is mostly focused on lore characters relevant to the First and Second Wars, but there are a few additional characters in the mix as well.

# How to test:

Look for the aforementioned characters in-game in their respective bookmarks, observe their custom DNA, and write down your thoughts, feedback, and suggestions if there are any.

If you need to have a better look at some of the characters, you can copy their DNA from `common\dna_data\wc_<race>_dna.txt` and paste it into either the Ruler Designer of the [Debug Portrait Editor](https://ck3.paradoxwikis.com/Console_commands#Debug_mode).

When copying DNA, only copy this part for it to work:

<details><summary>Click to expand</summary>

```
	portrait_info = {
		genes = {
			hair_color={ 24 243 28 234 }
			skin_color={ 28 30 48 41 }
			eye_color={ 152 91 35 221 }
			gene_chin_forward={ "chin_forward_neg" 146 "chin_forward_neg" 117 }
			gene_chin_height={ "chin_height_pos" 129 "chin_height_pos" 129 }
			gene_chin_width={ "chin_width_neg" 196 "chin_width_neg" 121 }
			gene_eye_angle={ "eye_angle_neg" 138 "eye_angle_neg" 118 }
			gene_eye_depth={ "eye_depth_neg" 114 "eye_depth_neg" 114 }
			gene_eye_height={ "eye_height_pos" 100 "eye_height_pos" 147 }
			gene_eye_distance={ "eye_distance_neg" 131 "eye_distance_neg" 97 }
			gene_eye_shut={ "eye_shut_pos" 113 "eye_shut_pos" 142 }
			gene_forehead_angle={ "forehead_angle_neg" 105 "forehead_angle_neg" 105 }
			gene_forehead_brow_height={ "forehead_brow_height_pos" 140 "forehead_brow_height_pos" 127 }
			gene_forehead_roundness={ "forehead_roundness_pos" 174 "forehead_roundness_pos" 174 }
			gene_forehead_width={ "forehead_width_pos" 172 "forehead_width_pos" 172 }
			gene_forehead_height={ "forehead_height_neg" 108 "forehead_height_neg" 108 }
			gene_head_height={ "head_height_pos" 160 "head_height_pos" 167 }
			gene_head_width={ "head_width_neg" 114 "head_width_neg" 89 }
			gene_head_profile={ "head_profile_neg" 82 "head_profile_neg" 118 }
			gene_head_top_height={ "head_top_height_neg" 95 "head_top_height_neg" 95 }
			gene_head_top_width={ "head_top_width_pos" 169 "head_top_width_pos" 127 }
			gene_jaw_angle={ "jaw_angle_pos" 95 "jaw_angle_pos" 164 }
			gene_jaw_forward={ "jaw_forward_neg" 115 "jaw_forward_neg" 115 }
			gene_jaw_height={ "jaw_height_neg" 167 "jaw_height_neg" 125 }
			gene_jaw_width={ "jaw_width_neg" 126 "jaw_width_neg" 52 }
			gene_mouth_corner_depth={ "mouth_corner_depth_pos" 157 "mouth_corner_depth_pos" 157 }
			gene_mouth_corner_height={ "mouth_corner_height_pos" 89 "mouth_corner_height_pos" 135 }
			gene_mouth_forward={ "mouth_forward_pos" 129 "mouth_forward_pos" 129 }
			gene_mouth_height={ "mouth_height_pos" 80 "mouth_height_pos" 136 }
			gene_mouth_width={ "mouth_width_pos" 170 "mouth_width_pos" 133 }
			gene_mouth_upper_lip_size={ "mouth_upper_lip_size_neg" 112 "mouth_upper_lip_size_neg" 112 }
			gene_mouth_lower_lip_size={ "mouth_lower_lip_size_neg" 196 "mouth_lower_lip_size_neg" 123 }
			gene_mouth_open={ "mouth_open_neg" 0 "mouth_open_neg" 44 }
			gene_neck_length={ "neck_length_pos" 127 "neck_length_pos" 127 }
			gene_neck_width={ "neck_width_neg" 113 "neck_width_neg" 102 }
			gene_bs_cheek_forward={ "cheek_forward_pos" 209 "cheek_forward_pos" 28 }
			gene_bs_cheek_height={ "cheek_height_neg" 255 "cheek_height_pos" 24 }
			gene_bs_cheek_width={ "cheek_width_pos" 43 "cheek_width_neg" 20 }
			gene_bs_ear_angle={ "ear_angle_pos" 47 "ear_angle_neg" 27 }
			gene_bs_ear_inner_shape={ "ear_inner_shape_pos" 20 "ear_inner_shape_pos" 20 }
			gene_bs_ear_bend={ "ear_lower_bend_pos" 13 "ear_lower_bend_pos" 13 }
			gene_bs_ear_outward={ "ear_outward_pos" 77 "ear_outward_neg" 204 }
			gene_bs_ear_size={ "ear_size_pos" 183 "ear_size_neg" 46 }
			gene_bs_eye_corner_depth={ "eye_corner_depth_pos" 151 "eye_corner_depth_neg" 88 }
			gene_bs_eye_fold_shape={ "eye_fold_shape_neg" 17 "eye_fold_shape_pos" 21 }
			gene_bs_eye_size={ "eye_size_pos" 91 "eye_size_pos" 171 }
			gene_bs_eye_upper_lid_size={ "eye_upper_lid_size_pos" 91 "eye_upper_lid_size_neg" 43 }
			gene_bs_forehead_brow_curve={ "forehead_brow_curve_neg" 122 "forehead_brow_curve_neg" 53 }
			gene_bs_forehead_brow_forward={ "forehead_brow_forward_pos" 234 "forehead_brow_forward_pos" 26 }
			gene_bs_forehead_brow_inner_height={ "forehead_brow_inner_height_neg" 195 "forehead_brow_inner_height_pos" 96 }
			gene_bs_forehead_brow_outer_height={ "forehead_brow_outer_height_pos" 121 "forehead_brow_outer_height_neg" 80 }
			gene_bs_forehead_brow_width={ "forehead_brow_width_pos" 177 "forehead_brow_width_pos" 51 }
			gene_bs_jaw_def={ "jaw_def_pos" 129 "jaw_def_neg" 159 }
			gene_bs_mouth_lower_lip_def={ "mouth_lower_lip_def_pos" 17 "mouth_lower_lip_def_pos" 17 }
			gene_bs_mouth_lower_lip_full={ "mouth_lower_lip_full_pos" 99 "mouth_lower_lip_full_neg" 30 }
			gene_bs_mouth_lower_lip_pad={ "mouth_lower_lip_pad_pos" 163 "mouth_lower_lip_pad_pos" 31 }
			gene_bs_mouth_lower_lip_width={ "mouth_lower_lip_width_pos" 51 "mouth_lower_lip_width_pos" 51 }
			gene_bs_mouth_philtrum_def={ "mouth_philtrum_def_pos" 163 "mouth_philtrum_def_pos" 44 }
			gene_bs_mouth_philtrum_shape={ "mouth_philtrum_shape_neg" 24 "mouth_philtrum_shape_neg" 24 }
			gene_bs_mouth_philtrum_width={ "mouth_philtrum_width_pos" 47 "mouth_philtrum_width_pos" 47 }
			gene_bs_mouth_upper_lip_def={ "mouth_upper_lip_def_pos" 123 "mouth_upper_lip_def_pos" 123 }
			gene_bs_mouth_upper_lip_full={ "mouth_upper_lip_full_pos" 11 "mouth_upper_lip_full_pos" 11 }
			gene_bs_mouth_upper_lip_profile={ "mouth_upper_lip_profile_pos" 45 "mouth_upper_lip_profile_pos" 45 }
			gene_bs_mouth_upper_lip_width={ "mouth_upper_lip_width_pos" 125 "mouth_upper_lip_width_pos" 125 }
			gene_bs_nose_forward={ "nose_forward_pos" 161 "nose_forward_neg" 27 }
			gene_bs_nose_height={ "nose_height_neg" 50 "nose_height_pos" 109 }
			gene_bs_nose_length={ "nose_length_pos" 38 "nose_length_pos" 38 }
			gene_bs_nose_nostril_height={ "nose_nostril_height_pos" 47 "nose_nostril_height_neg" 13 }
			gene_bs_nose_nostril_width={ "nose_nostril_width_pos" 0 "nose_nostril_width_neg" 33 }
			gene_bs_nose_profile={ "nose_profile_hawk" 95 "nose_profile_pos" 56 }
			gene_bs_nose_ridge_angle={ "nose_ridge_angle_neg" 30 "nose_ridge_angle_neg" 30 }
			gene_bs_nose_ridge_width={ "nose_ridge_width_neg" 80 "nose_ridge_width_neg" 34 }
			gene_bs_nose_size={ "nose_size_pos" 48 "nose_size_pos" 48 }
			gene_bs_nose_tip_angle={ "nose_tip_angle_neg" 72 "nose_tip_angle_neg" 4 }
			gene_bs_nose_tip_forward={ "nose_tip_forward_pos" 110 "nose_tip_forward_pos" 8 }
			gene_bs_nose_tip_width={ "nose_tip_width_pos" 7 "nose_tip_width_pos" 87 }
			face_detail_cheek_def={ "cheek_def_02" 0 "cheek_def_02" 65 }
			face_detail_cheek_fat={ "cheek_fat_04_pos" 0 "cheek_fat_04_pos" 24 }
			face_detail_chin_cleft={ "chin_cleft" 2 "chin_cleft" 2 }
			face_detail_chin_def={ "chin_def" 255 "chin_def" 217 }
			face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 255 "eye_lower_lid_def" 131 }
			face_detail_eye_socket={ "eye_socket_01" 255 "eye_socket_02" 45 }
			face_detail_nasolabial={ "nasolabial_03" 132 "nasolabial_02" 35 }
			face_detail_nose_ridge_def={ "nose_ridge_def_pos" 14 "nose_ridge_def_pos" 14 }
			face_detail_nose_tip_def={ "nose_tip_def" 3 "nose_tip_def" 3 }
			face_detail_temple_def={ "temple_def" 181 "temple_def" 181 }
			expression_brow_wrinkles={ "brow_wrinkles_04" 255 "brow_wrinkles_01" 57 }
			expression_eye_wrinkles={ "eye_wrinkles_03" 255 "eye_wrinkles_01" 82 }
			expression_forehead_wrinkles={ "forehead_wrinkles_01" 0 "forehead_wrinkles_01" 39 }
			expression_other={ "cheek_wrinkles_left_01" 0 "cheek_wrinkles_left_01" 127 }
			complexion={ "complexion_1" 78 "complexion_6" 78 }
			gene_height={ "normal_height" 131 "normal_height" 137 }
			gene_bs_body_type={ "body_fat_head_fat_low" 136 "body_fat_head_fat_low" 136 }
			gene_bs_body_shape={ "body_shape_triangle_half" 124 "body_shape_pear_full" 35 }
			gene_bs_bust={ "bust_clothes" 92 "bust_shape_4_full" 92 }
			gene_age={ "old_2" 0 "old_2" 224 }
			gene_eyebrows_shape={ "avg_spacing_avg_thickness" 163 "avg_spacing_low_thickness" 101 }
			gene_eyebrows_fullness={ "layer_2_high_thickness" 161 "layer_2_avg_thickness" 93 }
			gene_body_hair={ "body_hair_sparse" 0 "body_hair_sparse" 139 }
			hairstyles={ "all_hairstyles" 94 "western_hairstyles" 117 }
			beards={ "all_beards" 228 "western_beards" 127 }
			eye_accessory={ "normal_eyes" 230 "normal_eyes" 230 }
			teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
			eyelashes_accessory={ "normal_eyelashes" 91 "normal_eyelashes" 91 }
			gene_bs_ear_lenght={ "floppy_ear_length" 0 "floppy_ear_length" 0 }
			special_eyebrows={ "no_eyebrows" 44 "no_eyebrows" 44 }
			special_eyes={ "no_special_eyes" 0 "no_special_eyes" 0 }
			gene_bs_mouth_tusks={ "mouth_fangs" 0 "mouth_fangs" 0 }
			static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
			gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
			gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
		}
	}
	enabled=yes
}
```

</details>

In other words, **_do not_** copy the name at the top, otherwise, the game won't recognize the DNA.

❗ **Important**: Keep the limits of the game's Ruler Designer in mind when you give your feedback.

### CODE REVIEW

**Comment by ercarp** [comment]
Go'el/Thrall is currently crazy tall, which I don't believe is intended. Will be fixed.

File: common/dna_data/wc_orc_dna.txt [hunk 1]
```diff
@@ -0,0 +1,825 @@
...
+ 		face_detail_nasolabial={ "nasolabial_01" 237 "nasolabial_01" 229 }
+ 		face_detail_nose_ridge_def={ "nose_ridge_def_neg" 0 "nose_ridge_def_neg" 12 }
+ 		face_detail_nose_tip_def={ "nose_tip_def" 0 "nose_tip_def" 23 }
+ 		face_detail_temple_def={ "temple_def" 0 "temple_def" 229 }
+ 		expression_brow_wrinkles={ "brow_wrinkles_04" 255 "brow_wrinkles_03" 202 }
+ 		expression_eye_wrinkles={ "eye_wrinkles_03" 193 "eye_wrinkles_01" 193 }
+ 		expression_forehead_wrinkles={ "forehead_wrinkles_03" 255 "forehead_wrinkles_01" 255 }
+ 		expression_other={ "cheek_wrinkles_both_01" 0 "cheek_wrinkles_both_01" 209 }
+ 		complexion={ "complexion_7" 185 "complexion_2" 88 }
+ 		gene_height={ "normal_height" 255 "normal_height" 141 }
```
Comment (zumbak04) [inline common/dna_data/wc_orc_dna.txt:685]: Thrall is a giant.
Also, I think his skin is too dark.
Also, I would adjust his chin to be more orcish.
<details>
<summary>Click to expand</summary>

image
image

</details>

File: common/dna_data/wc_orc_dna.txt [hunk 2]
```diff
@@ -0,0 +1,825 @@
...
+ 		special_eyebrows={ "no_eyebrows" 111 "no_eyebrows" 111 }
+ 		special_eyes={ "no_special_eyes" 255 "no_special_eyes" 0 }
+ 		gene_bs_mouth_tusks={ "mouth_tusks_pos" 255 "mouth_tusks_pos" 255 }
+ 		static_tusks={ "orcish_tusks" 127 "orcish_tusks" 193 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 255 }
+		}
+	}
+	enabled=yes
+}
+guldan_dna = { #by Garthor13
```
Comment (zumbak04) [inline common/dna_data/wc_orc_dna.txt:119]: Gul'dan chin needs rework.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/dna_data/wc_orc_dna.txt [hunk 3]
```diff
@@ -0,0 +1,825 @@
+﻿### Orcs
+blackhand_dna = {
```
Comment (zumbak04) [inline common/dna_data/wc_orc_dna.txt:2]: Blackhand has too rough and fatless cheeks.
Also, his profile and chin it too forward.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/dna_data/wc_human_dna.txt [hunk 4]
```diff
@@ -0,0 +1,2609 @@
...
+			gene_bs_mouth_tusks={ "mouth_tusks_pos" 0 "mouth_tusks_pos" 0 }
+			static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+			gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+			clothes={ "western_war_nobility_clothes" 0 "most_clothes" 0 }
+		}
+	}
+	enabled=yes
+}
+
+arthas_menethil_dna_2 = { # by ercarp
```
Comment (zumbak04) [inline common/dna_data/wc_human_dna.txt:2491]: I like `arthas_menethil_dna_2` more. The other one is too chunky and feminine.

**Review by TheGroxEmpire — COMMENTED** [review]
I could see all Custom DNA applied in game, the age triggers are also working for all of them.
Suffice to say the Custom DNA for these characters are amazing and really accurate. They are _that_ good that I only found one issue on them:

- @Garthor19 Tirion Fordring was depicted as having short and brown hairs before his exiles (https://wow.gamepedia.com/File:TirionEitrigg.JPG). Currently he has white hair on the game on First War start date:
<details>

image
image
</details>

**Review by TheDrPibb1999 — APPROVED** [review]
I agree completely with TheGroxEmpire, Tirion being 33 with grey hair is a bit off, lorewise, and just how it would be in real life lol. but as a first issue, I had to issues with any of the DNA aging and they all looked amazing, especially that lor'themar whoever made that :o

Tirion in the second war comics as another source of his brown hair
TirionAshbringerComic

File: common/dna_data/wc_orc_dna.txt (hunk 1 above)
Comment (ercarp) [reply common/dna_data/wc_orc_dna.txt:685]: Fixed in fd9a9d95a8b6b2b1a8093df114ff40525820457e.

File: common/dna_data/wc_orc_dna.txt (hunk 3 above)
Comment (ercarp) [reply common/dna_data/wc_orc_dna.txt]: Fixed in 5bebd28b9fcc5a0aee20ad54fce0dfcd03d2b6cb.

File: common/dna_data/wc_orc_dna.txt (hunk 2 above)
Comment (ercarp) [reply common/dna_data/wc_orc_dna.txt:239]: Fixed in b5c2fdbd8f633dd6cbc70f98391dc797c039286f.

**Comment by ercarp** [comment]
Tirion's hair fixed in 009e6273f7346a87891318239b641d9d29018c78.

**Review by TheDrPibb1999 — APPROVED** [review]
All changes to Go'el, Blackhand, Gul'dan, and Tirion all LGTM. I think the red eyes for Gul'dan is better for appearances and as a distinguishing mark imo. good work

File: common/dna_data/wc_human_dna.txt [hunk 5]
```diff
@@ -0,0 +1,2870 @@
...
+			gene_bs_mouth_tusks={ "mouth_tusks_pos" 0 "mouth_tusks_pos" 0 }
+			static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+			gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+nathanos_marris_undead_dna = { #by ercarp
```
Comment (zumbak04) [inline common/dna_data/wc_human_dna.txt:841]: Are we using it?

File: common/dna_data/wc_orc_dna.txt [hunk 6]
```diff
@@ -0,0 +1,1195 @@
...
+ 		gene_bs_mouth_tusks={ "mouth_tusks_pos" 255 "mouth_tusks_pos" 255 }
+ 		static_tusks={ "orcish_tusks" 127 "orcish_tusks" 193 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 255 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+blackhand_dna_2 = { # base by Garthor13, adjusted by ercarp
```
Comment (zumbak04) [inline common/dna_data/wc_orc_dna.txt:121]: He has no hairs and beard. Intented?
<details>
<summary>Click to expand</summary>

image

</details>

File: common/dna_data/wc_orc_dna.txt [hunk 7]
```diff
@@ -0,0 +1,1195 @@
...
+ 		static_tusks={ "orcish_tusks" 127 "orcish_tusks" 193 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 255 }
+ 		gene_scars_battle={ "scars_battle" 255 "scars_battle" 0 }
+ 		special_headgear_eye_patch={ "eye_patch" 127 "eye_patch" 0 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+orgrim_doomhammer_dna = { # by Garthor13
```
Comment (zumbak04) [inline common/dna_data/wc_orc_dna.txt:602]: He has a very weak chin compared to other orcs.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/dna_data/wc_orc_dna.txt [hunk 8]
```diff
@@ -0,0 +1,1195 @@
...
+			gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 255 }
+			gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
+			clothes={ "western_bedchamber" 89 "most_clothes" 0 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+kilrogg_dna = { #by Garthor13
```
Comment (zumbak04) [inline common/dna_data/wc_orc_dna.txt:482]: I would make his chin is less forward.
<details>
<summary>Click to expand</summary>

image

</details>

He lacks hairs and beards.

<details>
<summary>Click to expand</summary>

image

</details>

File: common/dna_data/wc_orc_dna.txt [hunk 9]
```diff
@@ -0,0 +1,1195 @@
...
+ 		gene_bs_mouth_tusks={ "mouth_tusks_pos" 255 "mouth_tusks_pos" 255 }
+ 		static_tusks={ "orcish_tusks" 237 "orcish_tusks" 237 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 255 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+thrall_dna_2 = { # base by Garthor13, adjusted by ercarp
```
Comment (zumbak04) [inline common/dna_data/wc_orc_dna.txt:958]: He lacks hair and beards.
<details>
<summary>Click to expand</summary>



</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/dna_data/wc_human_dna.txt (hunk 5 above)
Comment (ercarp) [reply common/dna_data/wc_human_dna.txt:841]: Because of the custom undead portrait modifier for Nathanos, no, I don't think so.

File: common/dna_data/wc_elf_dna.txt [hunk 10]
```diff
@@ -0,0 +1,484 @@
...
+			gene_bs_mouth_tusks={ "mouth_tusks_pos" 0 "mouth_tusks_pos" 0 }
+			static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+			gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+sylvanas_windrunner_dna = { # by ercarp
```
[the remark this answers is not in the document]
Comment (ercarp) [reply common/dna_data/wc_elf_dna.txt:363]: She does look pretty odd. Not sure what happened there, but it could also just be the animation she's using since those accentuate certain facial features.

File: common/dna_data/wc_human_dna.txt (hunk 5 above)
Comment (zumbak04) [reply common/dna_data/wc_human_dna.txt:841]: So could you remove it?

File: common/dna_data/wc_elf_dna.txt (hunk 10 above)
[the remark this answers is not in the document]
Comment (ercarp) [reply common/dna_data/wc_elf_dna.txt:363]: Looks fine on my end, but I was able to replicate the issue using a different idle animation.

<details><summary>Click to expand</summary>

image

</details>

File: common/dna_data/wc_orc_dna.txt (hunk 7 above)
Comment (ercarp) [reply common/dna_data/wc_orc_dna.txt:724]: Fixed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.

File: common/dna_data/wc_orc_dna.txt (hunk 8 above)
Comment (ercarp) [reply common/dna_data/wc_orc_dna.txt:482]: Fixed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.

File: common/dna_data/wc_orc_dna.txt (hunk 9 above)
Comment (ercarp) [reply common/dna_data/wc_orc_dna.txt:1200]: Fixed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.

File: common/dna_data/wc_orc_dna.txt (hunk 6 above)
Comment (ercarp) [reply common/dna_data/wc_orc_dna.txt:121]: Not intended. Fixed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.

File: common/dna_data/wc_human_dna.txt (hunk 5 above)
Comment (ercarp) [reply common/dna_data/wc_human_dna.txt]: Removed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.

File: common/dna_data/wc_elf_dna.txt [hunk 11]
```diff
@@ -0,0 +1,484 @@
...
+			gene_bs_mouth_tusks={ "mouth_tusks_pos" 0 "mouth_tusks_pos" 0 }
+			static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+			gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+anasterian_dna = { # by ercarp
```
[the remark this answers is not in the document]
Comment (ercarp) [reply common/dna_data/wc_elf_dna.txt:244]: Fixed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by TheDrPibb1999 — APPROVED** [review]
All things addressed in 1.3 switch seems fixed to me! Great stuff!

**Review by KingofMushan — COMMENTED** [review]
Tested with multiple hairstyles, age ranges, and traits that change appearance (Wounded, sick, etc.)
Magni: No Issue
Lor'themar: No Issue
Kael'thas: No Issue
Anasterian: No Issue
Sylvanas: Small gaps between torso and neck when the "Spindly" trait is applied.
Daelin: Several beards cause minor to major clipping with the upper lip. 
Garithos: No Issue
Antonidas: Strange Eye Color, but I can't find a source to suggest it's incorrect. I'd suggest making them more vibrantly yellow if we're going with the yellow color regardless.
Khadgar: Similar beard clipping issues as Daelin. 
Thoras: Minor clipping issues between certain beards and upper lip.
Llane: Strangely pale, and hair color seems strange. If Varian has dark brown/black hair and Anduin has blonde, why would Llane have a kind of ginger/blond?
Nathanos: No Issue
Turalyon: No Issue
Anduin Wrynn: Strangely tall forehead, otherwise no issue
Varian: No Issue
Anduin Lothar: Same minor beard clipping issues as Thoras
Medivh: Should have special "magic" eyes like Antonidas.
Katherine: Same gap issues as Sylvanas when "Spindly" is applied.
Terenas: No Issue
Arthas: Neck seems too large, otherwise no issue

I can finish up the others when I have a bit more time, this is all I could get through in one sitting. Hope I'm not being too nitpicky, but I'm nothing if not thorough.

**Review by TheGroxEmpire — APPROVED** [review]
Appeared to be correctly ported to 1.3, LGTM!
